blueprints/ember-web-app/index.js

```diff
@@ -15,6 +15,19 @@ module.exports = {
     };
   },
 
+  fileMapTokens() {
+    const absPath = this.project.configPath(); // I can use this, but relative is better
```

Comment:
Why is relative better?

Would it make sense to describe the reason in comment instead?

Reply:
😂Fragments of a reminder for myself when I got distracted. Anyway...

The file path is actually logged to the console when you generate the file. It can get long and wouldn’t be very useful to see the full path. 

I can add a better explanation